### PR TITLE
[RLlib] Re-establish dashboard performance tests (renamed to `dashboard_tests` for more clarity).

### PR DESCRIPTION
**Pull request by sven1977**
Re-establish dashboard performance tests.

These tests seem to have gotten de-activated during a previous release tests re-organization.

* merged into existing learning_tests
* make learning_tests nightly
* reduce CI-learning test load by retiring those algos/envs that are already covered anyways by the nightly learning tests

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by maxpumperla — APPROVED** [review]
LGTM, not sure if Kai has any objections here.

**Review by krfricke — COMMENTED** [review]
LGTM, let's wait for CI to pass. Also did you kick off a release test run from the pipeline?

**Pull request merged**
